cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1000,13 +1000,14 @@ def _validate_obsm(self):
                     self.errors.append(
                         f"Suffix for embedding key in 'adata.obsm' {key} does not match the regex pattern {regex_pattern}."
                     )
-            else:
+            elif key.lower() != "spatial":
```

Comment:
diff is from https://github.com/chanzuckerberg/single-cell-curation/issues/827 merged into main